src/notte/sdk/endpoints/persona.py

```diff
@@ -35,6 +46,9 @@ class PersonaClient(BaseClient):
     SMS_READ = "{persona_id}/sms/read"
     CREATE_NUMBER = "{persona_id}/create-number"
     CREATE_PERSONA = "create"
+    ADD_CREDENTIALS = "{persona_id}/add-credentials"
```

Comment:
Can we create a shared prefix `secrets orcredentials`  `personas/{id}/secrets/add` ? To better organise the endpoints ?